src/js/inventory/inventoryStyles.js

```diff
@@ -1,9 +0,0 @@
-import '@patternfly/patternfly/components/Toolbar/toolbar.css';
```

Comment:
I am a bit sceptical about these changes, but we can only see it fail once we deploy that. Let's see on ci-beta, afterall this env is supposed to be for testing integrations.

Reply:
All of this should be imported with the components :crossed_fingers: . We will see, we can always put it back.